CenterForOpenScience/ember-osf-web
# [ENG-1406] [Feature] Remove edit contributors link on overview page

- Ticket: https://openscience.atlassian.net/browse/ENG-1406
- Feature flag: n/a

## Purpose

The contributors widget on the overview page currently shows the `Edit contributors` link regardless of what permissions the user has. Because it's on the overview page, the edit link shouldn't even show up, to match the contributors widget on the sidebar too.

## Summary of Changes

- Add conditional to not show the edit link on the overview page

## Side Effects

This shouldn't have any side effects

## QA Notes

This was an edit to the contributors widget in schema blocks, which is used in only two places: the overview page, and in the actual registration submission form. It shouldn't affect anything outside of that scope, but those two instances should probably be checked to make sure the link is showing up when it needs to.
